assets/data-port/import/data/actions.js

```diff
@@ -34,13 +35,19 @@ export const fetchFromAPI = ( request ) => ( {
 
 /**
  * Fetch importer data action creator.
+ *
+ * @param {string} jobId Job identifier.
  */
-export function* fetchImporterData() {
+export function* fetchImporterData( jobId ) {
```

Comment:
Looks like this is not actually used with a `jobId` supplied (as the polling is just calling fetchFromAPI directly).
Also WDYT about naming this and related things to something like `getJobState` or `getCurrentJobState`?

Reply:
Renamed to `getCurrentJobState` b294057081da8e38dcc9dd1f5843e51aaeb9b655

Reply:
Renamed to `fetchCurrentJobState` in 85eb989c0f239af74e6e47be7aca1ef2615b817a based on @renatho's feedback.